README.md

```diff
@@ -27,7 +27,7 @@ Submit your requirement as an issue to [https://github.com/omise/omise-magento/i
 
 The steps below shows how to install the module manually.
 
-1. Download and extract the zip file from [Omise-Magento](https://github.com/omise/omise-magento/archive/v1.21.zip) to your `local machine` (or directly to your server).
+1. Download and extract the zip file from [Omise-Magento](https://github.com/omise/omise-magento/archive/v1.22.zip) to your `local machine` (or directly to your server).
```

Comment:
To make it easier on ourselves, can we just point people to get the latest release on 1.x branch on this page.  Then we don't have to remember to update this line

https://github.com/omise/omise-magento/releases

Reply:
We have releases listed together for Magento 1 and 2. In my opinion, giving link to latest release makes more easier for users instead of link to release list as they have to find latest release on their own.

Reply:
OK, great, that makes sense.